git_patrol.py

```diff
@@ -310,14 +335,29 @@ def __init__(self):
     commands: GitPatrolCommands object used to execute external commands.
     loop: A reference to the asyncio event loop in use.
     db: A GitPatrolDb object used for database operations.
-    target: Git Patrol config target information.
+    config_path: Path to files referenced by the target configuration.
+    target_config: Git Patrol config target information.
     offset: Starting offset time in seconds.
     interval: Time in seconds to wait between poll attempts.
   Returns:
     Nothing. Loops forever.
   """
   alias = target_config['alias']
   url = target_config['url']
+  ref_filters = []
+  if 'ref_filters' in target_config:
+    ref_filters = target_config['ref_filters']
+
+  # Validate target configuration.
+  if len(ref_filters) > MAX_REF_FILTERS:
+    logger.error('%s: too many ref filters provided', alias)
+    return
+  if len(ref_filters) > 0:
+    validate_tasks = [git_check_ref_filter(commands, f) for f in ref_filters]
+    ref_filters_ok = await asyncio.gather(*validate_tasks)
```

Comment:
depending on how this function handles an empty list, I think it's logically equivalent to remove the `len(ref_filters) > 0` check

Reply:
Yep, turns out it is ok to pass an empty list to `asyncio.gather()`. It's also useful that `all([])` is `True`. Done.